src/index.js

```diff
@@ -76,10 +76,18 @@ class Logentries extends stream.Subscriber {
       const minLogLevel = this.getMinLogLevel(this.settings, this.name);
       if (this.logLevels.get(messageLevel) >= this.logLevels.get(minLogLevel)) {
         const prefix = message.getPrefix(this.settings);
-        const messageText = !prefix.isEmpty ?
-          `[${prefix.timestamp}${prefix.environment}${prefix.logLevel}${prefix.reqId}]${content.text}` :
-          content.text;
-        this.winston.log(messageLevel, messageText, content.meta);
+        let prefixText = !prefix.isEmpty ?
+          `[${prefix.timestamp}${prefix.environment}${prefix.logLevel}${prefix.reqId}] ` : '';
+        // if prefix contains these props, then caller module prefix was configured by settings/env
+        if ({}.hasOwnProperty.call(prefix, 'module') &&
+            {}.hasOwnProperty.call(prefix, 'function') &&
+            {}.hasOwnProperty.call(prefix, 'project')) {
+          prefixText += `[${prefix.project}${prefix.module}${prefix.function}] `;
```

Comment:
shouldn't those be separated by separator or `:`?

Reply:
it is separated [internally](https://github.com/dial-once/node-logtify/blob/develop/src/modules/message.js#L114)

Reply:
ok